daemon/graphdriver/graphtest/graphtest_unix.go

```diff
@@ -310,15 +311,17 @@ func writeRandomFile(path string, size uint64) error {
 }
 
 // DriverTestSetQuota Create a driver and test setting quota.
-func DriverTestSetQuota(t *testing.T, drivername string) {
+func DriverTestSetQuota(t *testing.T, drivername string, required bool) {
 	driver := GetDriver(t, drivername)
 	defer PutDriver(t)
 
 	createBase(t, driver, "Base")
 	createOpts := &graphdriver.CreateOpts{}
 	createOpts.StorageOpt = make(map[string]string, 1)
 	createOpts.StorageOpt["size"] = "50M"
-	if err := driver.Create("zfsTest", "Base", createOpts); err != nil {
+	if err := driver.CreateReadWrite("zfsTest", "Base", createOpts); err == quota.ErrQuotaNotSupported && !required {
```

Comment:
Perhaps it makes sense to rename `"zfsTest"` to `drivername + "Test"` as it's no longer zfs-specific. Or just `"quotaTest"`...